waffle_configs/bedrock-prod.env

```diff
@@ -18,27 +18,12 @@ SWITCH_DATA_PRIVACY_DAY_2017=on
 SWITCH_DISABLE_SHA1_DOWNLOADS=off
 SWITCH_ELECTION_BUNDLE=on
 SWITCH_FIREFOX_ELECTION_BUILD=on
-SWITCH_EXPERIMENT_DE_HEARTS=off
-SWITCH_EXPERIMENT_HOME_2018=off
 SWITCH_EXPERIMENT_FIREFOX_NEW_FEATURES=on
 SWITCH_FIREFOX_DEV_FALL=on
 SWITCH_FIREFOX_CONCERT_SERIES=on
```

Comment:
Side note: It looks like this switch is still on, but would require some code changes in bedrock before it can be removed (so you're right not to remove it here 👍). Like the other concert switches, it may no longer be needed however. But I'm not quite clear.

Reply:
(it may also be totally fine to leave this switch in place and just remove it when we eventually retire that page)